scripts/zones/Arrapago_Remnants/mobs/Archaic_Rampart.lua

```diff
@@ -0,0 +1,110 @@
+-----------------------------------
+-- Area: Archaic Rampart
+--  MOB: Arrapago Remnants
+-----------------------------------
+mixins = {require("scripts/mixins/families/rampart")}
+local ID = require("scripts/zones/Arrapago_Remnants/IDs")
+require("scripts/globals/instance")
+require("scripts/globals/status")
+-----------------------------------
+function onMobSpawn(mob)
+    local instance = mob:getInstance()
+    mob:setLocalVar("swapTime", os.time())
+end
+
+function onMobFight(mob,target)
+    local instance = mob:getInstance()
+    local popTime = mob:getLocalVar("lastPetPop")
+    local POS = mob:getPos()
+    local PET1 = instance:getEntity(bit.band((mob:getID() +1), 0xFFF), dsp.objType.MOB)
+    local PET2 = instance:getEntity(bit.band((mob:getID() +2), 0xFFF), dsp.objType.MOB)
+    local PET3 = instance:getEntity(bit.band((mob:getID() +3), 0xFFF), dsp.objType.MOB)
+
+    if mob:getID() == 17080489 or  mob:getID() == 17080512 or mob:getID() == 17080553 then
```

Comment:
for mobs where they have the same name but do totally different things, its probably better to just give them 3 different scripts and change their names in the database

Reply:
we can also use their pool ID if that would be less `or`'s involved. But `Archaic_Rampart_identifierHere` works

Reply:
I'll admit that format is pretty messy, I was using ID ranges at some point. They do the same thing just have different amounts of adds. Some are ID +1/2, others are +1/2/3... I need a better way overall of doing these adds but in the meantime this was working for me.

Reply:
Changed them up so their names decide the amount of adds, removed all the `if ID == this` stuff.